src/db/models/Operator.js

```diff
@@ -24,7 +24,8 @@ module.exports = (sequelize, DataTypes) => {
       contact_representative_name: { type: DataTypes.STRING },
       contact_representative_role: { type: DataTypes.STRING },
       contact_representative_number: { type: DataTypes.STRING },
-      contact_representative_email: { type: DataTypes.STRING }
+      contact_representative_email: { type: DataTypes.STRING },
+      operator_uprn: { type: DataTypes.STRING }
```

Comment:
I think it's an integer, not a string - that was my mistake in the original API changes, sorry.

Reply:
It is an integer but FSA confirm treat as string